.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+This folder contains changesets, which are markdown files that hold two key bits of information: a
+version type (following semver), and change information to be added to a changelog. You can find the full documentation
```

Comment:
Nit: I think this would be a bit easier to read if the 2 elements listed here were formatted as a Markdown list.